blocked-edges/4.11.0-fc.0-cluster-verison-precondition-check.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.11.0-fc.0
+from: 4[.]10[.]([7-9]|1[0-4])[+].*
+url: https://bugzilla.redhat.com/show_bug.cgi?id=2072389#c11
+name: ClusterVersionPreconditionCheck 
+message: |-
+  The update will not proceed because the cluster version operator will not accept the new target release.
```

Comment:
Follow-up sentence like:

> We suggest updating to 4.10.15 or later versions (which contains the fix) before updating to 4.11.z.

to include the recommended next step from the impact statement?

Reply:
Fixed in https://github.com/openshift/cincinnati-graph-data/commit/8467cfc174899c716d46c309b721125b65b8ed88